### PR TITLE
Workflow Test

### PR DESCRIPTION
**Pull request by mastermind-mayhem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+* Pull requests are required for all changes to `main`.
+* Readable code is a priority, run `spotless` to ensure the
+  code you are changing is formatted correctly.
+  * Open the terminal window with CTRL+`
+  * Enter `./gradlew spotlessApply`
+  * Any changes should be included in your committed code.
+
+## Common Issues
+
+#### Pull request `checks have failed`
```
Comment (mathyourlife) [inline README.md]:
```suggestion
### Pull request `checks have failed`
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+
+If your pull request is blocked by a message like `All checks have failed` and `Merging is blocked`:
+
+* Open the pull request in the browser.
+* Click on "Details" which should bring you to the output of the failed check to figure out what needs to be fixed on your branch.
+
+#### Running gradle commands
```
Comment (mathyourlife) [inline README.md]:
```suggestion
### Running gradle commands
```

**Review by mathyourlife — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,34 @@
...
+
+To run a command like `./gradlew spotlessApply`,
```
Comment (mathyourlife) [inline README.md:30]: Is this command correct for windows laptops?  I think it's gradlew.bat instead

Comment (mastermind-mayhem) [reply README.md:30]: yeah it is

**Review by awesomeatpiano — APPROVED** [review]

**Pull request merged**